src/sdl2/event.rs

```diff
@@ -409,6 +409,7 @@ pub enum DisplayEvent {
     Orientation(Orientation),
     Connected,
     Disconnected,
+    Moved,
```

Comment:
Can you add a changelog entry (with `**BREAKING CHANGE**`) for this breaking change?